lib/TableUtils.lua

```diff
@@ -28,6 +28,14 @@ function TableUtils.Filter(source, handler) --: table, (element: any, key: numbe
 	return result
 end
 
+function TableUtils.Reduce(source, handler, init) --: <T>(table, (element: any, key: number | string => T), T) => T
```

Comment:
could do with some assertions

Reply:
I've now added tests, assertions pending ROBLOX-745 which is designed to inline assertions from the type definition

Reply:
What's the ETA on that? As long as it doesn't become a pipe dream that's ok